docs/Manual/Plugins/General/ConvertProfile.html

```diff
@@ -344,7 +344,7 @@ <h1 id="#convert_profile" class="anchor" >convert_profile</h1>
 particular software version. For example, Rekall requires a Linux Kernel profile
 to be able to analyze a memory image of the Linux kernel.</p>
 <p>The <code>convert_profile</code> plugin converts profiles other formats to the standard
-JSON format used by Relall. There are two main use cases:</p>
+JSON format used by Rekall. There are two main use cases:</p>
```

Comment:
There is no need to edit the html pages - they are generated from the .md pages automatically.